node/src/components/network/blocklist.rs

```diff
@@ -99,6 +101,9 @@ impl Display for BlocklistJustification {
                 write!(f, "sent a block that is invalid or unexpected ({})", error)
             }
             BlocklistJustification::DishonestPeer => f.write_str("dishonest peer"),
+            BlocklistJustification::SentTooManyFinalitySignatures {
+                max_allowed,
+            } => write!(f, "Peer sent too many finality signatures: maximum {max_allowed} signatures are allowed"),
```

Comment:
Nit:

for consistency:

```suggestion
            } => write!(f, "sent too many finality signatures: maximum {max_allowed} signatures are allowed"),
```

Reply:
Done